servicetalk-loadbalancer/src/main/java/io/servicetalk/loadbalancer/RoundRobinLoadBalancer.java

```diff
@@ -243,9 +244,7 @@ public void onNext(final ServiceDiscovererEvent<ResolvedAddress> event) {
                 // duplicates are not allowed
                 for (Host<ResolvedAddress, C> host : oldHostsTyped) {
                     if (host.address.equals(addr)) {
-                        if (!handleExpired) {
-                            LOGGER.warn("Duplicate ACTIVE event for host " + host + ". Duplicates are disallowed.");
```

Comment:
Not necessary to warn here, because duplicates will be expected when we remove deduplication logic from `DefaultServiceDiscoveryRetryStrategy`. Duplicates will be propagated if you retry SD events publisher. We can just silently ignore them or add trace/debug level logging later, if necessary.